rep-2000.rst

```diff
@@ -823,6 +823,119 @@ Package manager use for dependencies:
+| Ogre        |                                     1.10*                                     |         N/A         |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| OpenCV      |     4.5.4      |     4.5.4      |  3.4.6   |     4.2.0     |      4.5.1       |  4.1.0 / 3.2.0****  |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| OpenSSL     |     1.1.1l     |     1.1.1l     |  1.1.1g  |     1.1.1f    |      1.1.1i      | 1.1.1d / 1.1.1b**** |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Python      |     3.10.0     |     3.8.3      |  3.6.8   |     3.8.2     |      3.9.1       |  3.8.2 / 3.7.5****  |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Qt          |     5.15.2     |    5.12.10     |  5.12.5  |     5.12.3    |      5.15.2      | 5.14.1 / 5.12.5**** |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+|                              |                                   **Linux only**                                   |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| PCL         |     1.12.0     |      N/A       |  1.12.0  |      N/A      |      1.11.1      |       1.10.0        |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+|                                         **RMW DDS Middleware Providers**                                          |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Cyclone DDS |                                          0.8.x (Réplique)                                           |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Fast-DDS    |                                                2.4.x                                                |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Connext DDS |              5.3.1              |   N/A    |     5.3.1     |                  N/A                   |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+
+" * " means that this is not the upstream version (available on the official Operating System repositories) but a package distributed by OSRF or the community (package built and distributed on custom repositories).
+
+" ** " Rolling distributions will see multiple version changes of these dependencies during their lifetime.
```

Comment:
This sentence is confusing.  The reason is that this is talking about package managers that continually "roll" their package updates, without a stable API (like macOS homebrew).  It is *not* talking about `Rolling`, the ROS 2 distribution.  I think we should reword this somehow to make that clearer.

Reply:
How about the following?

> " ** " means that the dependency may see multiple version changes, because the dependency uses a package manager that continually updates the dependency without a stable API.

Reply:
If you like it, should I adjust elsewhere in the document, too, or I can make another PR?

Reply:
:+1: to the wording change you proposed.

:+1: to updating the rest of the document in a separate PR.

Reply:
> 👍 to the wording change you proposed.

https://github.com/ros-infrastructure/rep/pull/326/commits/ce8b91a5f82f28e471c3909443ccc32e6a35beb2.